client/lib/partner/utils.js

```diff
@@ -0,0 +1,16 @@
+import config from '@automattic/calypso-config';
+import { getCurrentPartner } from 'calypso/state/partner-portal/partner/selectors';
+
+export const isAgencyUser = ( partner ) => {
+	return partner?.partner_type === 'agency';
+};
+
+export const getShowAgencyDashboard = ( store ) => {
+	const state = store.getState();
+	let isAgency;
+	if ( config.isEnabled( 'jetpack/agency-dashboard' ) ) {
+		const partner = getCurrentPartner( state );
+		isAgency = isAgencyUser( partner );
+	}
+	return isAgency;
+};
```

Comment:
I suggest moving this to `client/state/selectors/`, which is the place where we put selectors.

Here are additional details: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/selectors/README.md

Reply:
Done ✅